.github/workflows/testing.yml

```diff
@@ -427,7 +447,7 @@ jobs:
         env:
           PY: py38
           GPU_TEST: ON
-          TI_WANTED_ARCHS: 'cpu,cuda,vulkan,opengl'
```

Comment:
we don't need cpu here because?

Reply:
Because CPU are tested in dedicated cpu tasks